Src/Framework/Jobs/Discover.py

```diff
@@ -244,15 +244,14 @@ def initialize(self):
         self._universe.initializeVelocitiesToTemperature(0.)
         self._velocities = ParticleVector(self._universe)
         self._velocities.array = self._hisfile["initial_velocities"]
-        self._universe.setVelocities(self._velocities)
         
         self._universe.foldCoordinatesIntoBox()
             
         # A MMTK trajectory is opened for writing.
         self._trajectory = Trajectory(self._universe, self.configuration['output_files']['files'][0], mode='w', comment=self._hisfile["title"])
 
         # A frame generator is created.
-        self._snapshot = SnapshotGenerator(self._universe, actions = [TrajectoryOutput(self._trajectory, ["all"], 0, None, 1)])
+        self._snapshot = SnapshotGenerator(self._universe, actions = [TrajectoryOutput(self._trajectory, ["configuration","velocities"], 0, None, 1)])

```

Comment:
This needs time as well.

Reply:
Sorry for the bug. I have been too fast. I should have double checked. Let's fix this.